src/components/Wizard/Patterns/WizardPattern.js

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { noop, propExists, Modal, Wizard, Icon, Button } from '../../../index';
+import WizardPatternBody from './WizardPatternBody';
+import { wizardStepShape } from './WizardPatternConstants';
+
+/**
+ * WizardPattern - the Wizard Pattern Body component.
+ */
+const WizardPattern = ({
+  steps,
+  activeStepIndex,
+  onStepChanged,
+  onNext,
+  onBack,
+  nextStepDisabled,
+  title,
+  loadingTitle,
+  loadingMessage,
+  show,
+  onHide,
+  onExited,
+  stepButtonsDisabled,
+  cancelText,
+  backText,
+  nextText,
+  closeText,
+  loading
+}) => {
+  const onBackClick = () => {
+    goToStep(Math.max(activeStepIndex - 1, 0));
+  };
+
+  const onNextClick = () => {
+    goToStep(Math.min(activeStepIndex + 1, steps.length - 1));
+  };
+
+  const getActiveStep = (relativeToIndex = activeStepIndex) =>
+    steps[relativeToIndex];
+
+  const getPrevStep = (relativeToIndex = activeStepIndex) =>
+    relativeToIndex > 0 && steps[relativeToIndex - 1];
+
+  const getNextStep = (relativeToIndex = activeStepIndex) =>
+    relativeToIndex < steps.length - 1 && steps[relativeToIndex + 1];
+
+  const goToStep = newStepIndex => {
+    const currentStep = steps[activeStepIndex];
+    if (shouldPreventGoToStep(newStepIndex)) return;
+    if (onStepChanged) onStepChanged(newStepIndex);
+    if (newStepIndex === activeStepIndex + 1) {
+      if (currentStep.onNext) currentStep.onNext();
+      if (onNext) onNext(newStepIndex);
+    }
+    if (newStepIndex === activeStepIndex - 1) {
+      if (onBack) onBack(newStepIndex);
+    }
+  };
+
+  const shouldPreventGoToStep = newStepIndex => {
+    const targetStep = steps[newStepIndex];
+    const stepBeforeTarget = getPrevStep(newStepIndex);
+
+    const preventExitActive = steps[activeStepIndex].preventExit;
+    const preventEnterTarget = propExists(targetStep, 'preventEnter')
+      ? targetStep.preventEnter
+      : stepBeforeTarget && stepBeforeTarget.isInvalid;
+    const nextStepClicked = newStepIndex === activeStepIndex + 1;
+
+    return (
+      preventExitActive ||
+      preventEnterTarget ||
+      (nextStepClicked && nextStepDisabled)
+    );
+  };
+
+  const onFirstStep = activeStepIndex === 0;
+  const onFinalStep = activeStepIndex === steps.length - 1;
+  const activeStepStr = (activeStepIndex + 1).toString();
+  const activeStep = getActiveStep();
+  const prevStepUnreachable =
+    onFirstStep || activeStep.preventExit || getPrevStep().preventEnter;
+  const nextStepUnreachable =
+    nextStepDisabled ||
+    activeStep.isInvalid ||
+    activeStep.preventExit ||
+    getNextStep().preventEnter;
+
+  return (
+    <Modal
```

Comment:
Note that priley86/patternfly-react@06bfb08 makes `Wizard` render `Modal` on its own.

This is aligned with the standard PatternFly wizard use case, which is "rendered as part of a modal".